server/plugin/command.go

```diff
@@ -589,22 +602,32 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 	mute.AddCommand(muteList)
 
 	settings := model.NewAutocompleteData("settings", "[setting] [value]", "Update your user settings")
-	setting := []model.AutocompleteListItem{{
-		HelpText: "Turn notifications on/off",
-		Item:     "notifications",
+
+	settingNotifications := model.NewAutocompleteData("notifications", "", "Turn notifications on/off")
+	settingValue := []model.AutocompleteListItem{{
+		HelpText: "Turn notifications on",
+		Item:     "on",
 	}, {
-		HelpText: "Turn reminders on/off",
-		Item:     "reminders",
+		HelpText: "Turn notifications off",
+		Item:     "off",
 	}}
-	settings.AddStaticListArgument("Setting to update", true, setting)
-	value := []model.AutocompleteListItem{{
-		HelpText: "Turn setting on",
+	settingNotifications.AddStaticListArgument("", true, settingValue)
+	settings.AddCommand(settingNotifications)
+
+	remainderNotifications := model.NewAutocompleteData("reminders", "", "Turn notifications on/off")
+	settingValue = []model.AutocompleteListItem{{
+		HelpText: "Turn reminders on",
 		Item:     "on",
 	}, {
-		HelpText: "Turn setting off",
+		HelpText: "Turn reminders off",
 		Item:     "off",
+	}, {
+		HelpText: "Turn reminders On - But only get reminders if any change occur",
+		Item:     "on-change",
```

Comment:
Can we use the constants e.g. `settingOnChange` here?

Reply:
sure